stellarisdashboard/game_info.py

```diff
@@ -1,15 +1,19 @@
 import json
+import re
 import logging
 
 logger = logging.getLogger(__name__)
 
 
 class NameRenderer:
     default_name = "Unknown name"
+    loc_re = None
```

Comment:
Why this class variable, I think only the one in `__init__` is actually used, right?

Reply:
I was probably thinking to not re-initialize it all the time.  I've pulled it out higher now